### DISCUSSION TITLE
Separate CLI from the diff library?

### DISCUSSION THREAD

**Discussion by AllanOricil** [scolladon/sfdx-git-delta#509] [Ideas]
@scolladon 
Maybe there could exist a VS Code extension that enable developers to deploy changes to salesforce environments when not using source tracking? I could create that if it doesn't exist already.

**Comment by scolladon** [comment]
I really like the idea 💯 
It could be another repo dedicated to it
Feel free to start this project if you feel eager to 👍

**Comment by scolladon** [comment]
Today the lib is [installable separatly](https://github.com/scolladon/sfdx-git-delta#use-the-module-in-your-own-node-application) from the CLI

That's a good idea to have an org, I add it to the next open source meeting agenda, thanks
